msteams-platform/tabs/how-to/authentication/auth-aad-sso.md

```diff
@@ -8,7 +8,7 @@ keywords: teams authentication SSO AAD single sign-on api
 
 # Single sign-on (SSO) support for tabs
 
-Users sign in to Microsoft Teams through their work, school, or Microsoft account that is Office 365, Outlook, and so on. You can take advantage by allowing a single sign-on to authorize your Teams tab or task module on desktop or mobile clients. If a user signs in once, they don't have to sign in again on another device as they're signed in automatically. Also, your access token is prefetched to improve performance and load times.
+When you sign in to Microsoft Teams through your work, school, or Microsoft account that is Office 365, Outlook, you can take the advantage by allowing a single sign on to authorize your Teams tab or task module on desktop or mobile clients. Once you are sign in, you don't have to sign in again on another device as you're signed in automatically. Also, your access token is prefetched to improve performance and load times.
```

Comment:
Please revert to original as You refers to developers and users are the end users.